pkg/clusterapi/aws/actuator.go

```diff
@@ -507,30 +500,70 @@ write_files:
   owner: 'root:root'
   permissions: '0640'
   content: |
-    openshift_group_type: %[1]s
+    openshift_group_type: {{ .NodeType }}{{ if .IsNode }}
```

Comment:
For the sake of readability, but the control statements on separate lines.

For example, put the `{{ if .IsNode }}` on a separate line and use `{{- if .IsNode }}` to strip out the whitespace to the left.

Reply:
Ooo that's a nice tip thanks.